packages/amplify_api/ios/Classes/FlutterApiErrorMessage.swift

```diff
@@ -19,4 +19,24 @@ enum FlutterApiErrorMessage: String {
     case MALFORMED = "AMPLIFY_REQUEST_MALFORMED"
     case QUERY_FAILED = "AMPLIFY_API_QUERY_FAILED"
     case MUTATE_FAILED = "AMPLIFY_API_MUTATE_FAILED"
+    case GET_FAILED = "AMPLIFY_API_GET_FAILED"
+    case PUT_FAILED = "AMPLIFY_API_PUT_FAILED"
+    case POST_FAILED = "AMPLIFY_API_POST_FAILED"
+    case DELETE_FAILED = "AMPLIFY_API_DELETE_FAILED"
+    case HEAD_FAILED = "AMPLIFY_API_HEAD_FAILED"
+    case PATCH_FAILED = "AMPLIFY_API_PATCH_FAILED"
+
+    
+    public static func stringToAPIRestError(methodName: String) -> FlutterApiErrorMessage{
```

Comment:
Rename to `getErrorForApi()` similar to and return a `String` similar to the Kotlin implementation.